cfme/tests/containers/test_containers_properties.py

```diff
@@ -40,7 +43,8 @@ def test_pods_properties_rel(provider, rel):
     the fields in the Properties table
     """
     navigate_to(Pod, 'All')
-    ui_pods = [r.name.text for r in list_tbl_pods.rows()]
+    ui_pods = sample([r.name.text for r in list_tbl_pods.rows()],
```

Comment:
since this  piece of code repeats 4 times, once for each table, it may be a good idea to give it a name and turning it into a name by defining a function

the first things that comes to mind for local usage right in the file is `name_samples_of(tbl)`